### PR TITLE
CargoOrderableCleansing

### PR DESCRIPTION
**Pull request by Snypehunter007**
A branch for removing, re-organizing, and altering the contents of crates order-able by Cargo.

Typically involves stripping crates that are too goofy and/or don't fit, altering existing crates contents to use stuff like modern uniforms, etc.

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: code/datums/supplypacks/medical.dm [hunk 1]
```diff
@@ -98,12 +98,10 @@
 	name = "Chief medical officer equipment"
 	contains = list(/obj/item/weapon/storage/belt/medical,
 					/obj/item/device/radio/headset/heads/smo,
-					/obj/item/clothing/under/rank/chief_medical_officer,
+					/obj/item/clothing/under/senior_medical_officer,
 					/obj/item/weapon/reagent_containers/hypospray/vial,
 					/obj/item/clothing/accessory/stethoscope,
 					/obj/item/clothing/glasses/hud/health,
-					/obj/item/clothing/suit/storage/toggle/labcoat/cmo,
```
Comment (NanakoAC) [inline code/datums/supplypacks/medical.dm:105]: why remove these labcoats?

File: code/datums/supplypacks/medical.dm [hunk 2]
```diff
@@ -135,50 +132,6 @@
 	containername = "\improper Medical Doctor equipment"
 	access = access_medical
 
-/decl/hierarchy/supply_pack/medical/chemistgear
```
Comment (NanakoAC) [inline code/datums/supplypacks/medical.dm:138]: why remove these gear packs?

File: code/datums/supplypacks/security.dm [hunk 3]
```diff
@@ -28,21 +28,6 @@
 	containername = "\improper Armor crate"
 	access = access_security
 
-/decl/hierarchy/supply_pack/security/tacticalarmor
```
Comment (NanakoAC) [inline code/datums/supplypacks/security.dm]: this one was fine, keep the armor packs generally

**Review by NanakoAC — CHANGES_REQUESTED** [review]
A few concerns, don't delete the armor packs please

**Review by ghost — CHANGES_REQUESTED** [review]
Place back the armor packs Nanako mentioned for now, I forgot that we'll be adding light armor pieces for crew to get their hands on, those packs will be the framework. After that should be good to go.

**Review by ghost — APPROVED** [review]

**Pull request merged**